test/CompanyService.Broker.UnitTests/FindDepartmentConsumerTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.CompanyService.Broker.Consumers;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.UnitTestKernel;
+using MassTransit;
+using MassTransit.Testing;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.CompanyService.Broker.UnitTests
+{
+    class FindDepartmentConsumerTests
+    {
+        private InMemoryTestHarness _harness;
+        private IRequestClient<IFindDepartmentsRequest> _requestClient;
+        private Mock<IDepartmentRepository> _repositoryMock;
+
+        private List<DbDepartment> _departments;
+
+        private const string _dontExistName = "dontExistName";
```

Comment:
Const fields should be have other naming

Reply:
how?

Reply:
UPPER_CASE or PascalCase, as i remember somewhere in project we already used UPPER_CASE, like DONT_EXIST_NAME

Reply:
I didn't find our rules

Reply:
outsourcing.nat.tepkom.ru:20080/redmine/projects/digital-office/wiki/Backend_NameConvention

Reply:
In word, use CamelCase